### PR TITLE
Misc: add logo, attribution, light font

### PR DESCRIPTION
**Pull request by chelshaw**
Some cleanup items. I decided to split the main parallax clouds into top/bottom so that they wouldn't interfere with the logo spacing:

Screenshot 2024-12-15 at 9 43 36 PM

### CODE REVIEW

File: src/lib/components/Logo.svelte [hunk 1]
```diff
@@ -0,0 +1,112 @@
+<script lang="ts">
+	export const { hex = '#303326' } = $props();
```
Comment (emikjackson) [inline src/lib/components/Logo.svelte]: nit: can we replace this with `var(--charcoal)`?

File: src/routes/+page.svelte [hunk 2]
```diff
@@ -105,6 +106,7 @@
 <svelte:window bind:innerWidth bind:innerHeight />
 
 <main onscroll={(e) => (scrollY = (e.target as HTMLElement)?.scrollTop)}>
+	<Logo hex="#FFFAF5" />
```
Comment (emikjackson) [inline src/routes/+page.svelte]: same for `var(--cream)`

File: src/lib/components/Logo.svelte [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+	</a>
+</div>
+
+<style>
+	div {
+		display: block;
+		position: absolute;
+		left: 30;
+		top: 30;
+		padding: 20px;
```
Comment (emikjackson) [inline src/lib/components/Logo.svelte:100]: I think we may need a z-index here so it's clickable above the clouds on the main page

File: src/routes/results/[results_string]/+page.svelte [hunk 4]
```diff
@@ -28,48 +29,58 @@
...
 		</div>
-	</div>
-</main>
+	</main>
+	<footer>
+		© 2024 The All We Can Save Project. Developers <a
+			href="https://github.com/chelshaw"
+			target="_blank"
+			rel="noopener nofollow">Chelsea Shaw</a
+		>, Emi Jackson, and Helen Li brought this interactive tool to life.
```
Comment (emikjackson) [inline src/routes/results/[results_string]/+page.svelte]: ooh nice, could you add my GH link too? `https://github.com/emikjackson` 🙏

**Review by emikjackson — APPROVED** [review]
sweet thanks! just left a few comments

File: src/routes/results/[results_string]/+page.svelte (hunk 4 above)
Comment (emikjackson) [reply src/routes/results/[results_string]/+page.svelte]: perhaps @li-helen wants hers added too?

Comment (emikjackson) [reply src/routes/results/[results_string]/+page.svelte]: ah I had missed your note on this on slack, thanks chelsea!

File: src/lib/components/Logo.svelte (hunk 1 above)
Comment (chelshaw) [reply src/lib/components/Logo.svelte]: I'm pretty sure that will only work within CSS. But I think I can finagle that to work here

File: src/routes/results/[results_string]/+page.svelte (hunk 4 above)
Comment (chelshaw) [reply src/routes/results/[results_string]/+page.svelte]: Went ahead and added both, we can always remove if @li-helen prefers a different link 😄

File: src/lib/components/Logo.svelte (hunk 3 above)
Comment (chelshaw) [reply src/lib/components/Logo.svelte:100]: good catch! resolved

**Review by li-helen — APPROVED** [review]

**Pull request merged**